compiler/AST/symbol.cpp

```diff
@@ -290,6 +298,51 @@ bool Symbol::noDocGen() const {
 }
 
 
+void Symbol::addSymExpr(SymExpr* se) {
+
+  // MPF 2016-11-08: Consider not tracking SymExprs
+  // that refer Symbols that have an immediate.
+  // The reason is that immediates are usually
+  // unique-ifiied and so the symbol for 3 would refer
+  // to all the uses of 3, and that probably isn't adding
+  // any value.
```

Comment:
Conceptually, I think that's a good thing to think about, but I'm not sure the complexity add would be worthwhile (which is I suspect why you didn't do it)